docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,172 @@
+# Autofill Manager
+
+The Tizen autofill framework allows more than one or more Autofill services to be installed on a Tizen device. Autofill manager is a module for managing the installed autofill services. Application developers can use autofill manager to get the current autofill service or change the autofill service.
+
```

Comment:
Please refer to the older content: https://developer.tizen.org/development/guides/native-application/media-and-camera/media-content-and-metadata

Heading must be in title case
![image](https://user-images.githubusercontent.com/35525967/53315331-82244b80-38ba-11e9-8afe-3638dabef6c9.png)

Reply:
I've updated heading with no gerund.